ambry-api/src/main/java/com/github/ambry/config/CompressionConfig.java

```diff
@@ -21,9 +21,9 @@
 public class CompressionConfig {
 
   // Boolean whether compression is enabled in PUT operation.
-  // TODO - Compression is disabled by default.  Need to enable after testing and verification.
+  // Compression is enabled by default.  It can be disabled using config.
   static final String COMPRESSION_ENABLED = "router.compression.enabled";
-  static final boolean DEFAULT_COMPRESSION_ENABLED = false;
+  static final boolean DEFAULT_COMPRESSION_ENABLED = true;
```

Comment:
I recommend doing this in two steps
First step is to enable V5 on PutRequest without enabling compression
Second step is to enable compression.

We can keep the most of the this PR but still make this default value to false.

Reply:
+1

Reply:
I have switch this back to "false".  It's okay to separate them to separate PRs.